### PR TITLE
Styling and lint formatting 

### PR DESCRIPTION
**Pull request by kellynauert**
Adjusted styling across the site

**Failed Tests:**
- TopNav is failing signout test, although it does correctly signout on both the web and mobile version. I changed the styling quite a bit, but I couldn't figure out what affected the test.
- TopNav is failing a test that checks that the menu is opening and closing correctly, even though it is. Again, I couldn't figure out what styling changes caused it to fail.
-  ProgramCollection is failing a test that expects text to match 'testuser'. I changed program cards to display "By {username}", so the 'By' is breaking the test. I wasn't sure if I should just add 'By' to the test expectation or not so I left it alone.

**Tests that may need to be added?**
- The appbar has a mobile version now that probably needs tests.
- The accordion on the courses page now tracks when the accordion is open or closed, and shows "Hide Course" or "Show Course" as needed.

### CODE REVIEW

**Comment by cabarnes** [comment]
When the user is over the free limit, the message does not fit in the new area
image

File: src/components/__tests__/ProgramCollection.test.js [hunk 1]
```diff
@@ -274,6 +274,30 @@ describe('The ProgramCollection component', () => {
...
+        user={testUser}
+        onProgramClick={onProgramClick}
+        onRemoveClick={onRemoveClick}
+        onUpdate={onUpdate}
+      />,
+    ).dive().dive().dive();
+
+    expect(wrapper.find('Styled(MuiBox)').at(2).prop('aria-owns')).toBe(undefined);
+    wrapper.find('Styled(MuiBox)').at(2).simulate('mouseenter', { target: { class: 'classes.flexitem3' } });
+    expect(wrapper.find('Styled(MuiBox)').at(2).prop('aria-owns')).toBe('mouse-over-popover');
```
Comment (hbradio) [inline src/components/__tests__/ProgramCollection.test.js:298]: Very nicely done on this test!
I'm going to copy what you did and add two lines to test that the popover disappears on `mouseleave`.

**Comment by cabarnes** [comment]
@kellynauert Did you intend for the `New Program` button to be multiple lines?
image

**Comment by kellynauert** [comment]
> @kellynauert Did you intend for the `New Program` button to be multiple lines?
It should always be one line, I will fix it.

**Review by cabarnes — APPROVED** [review]
Great job @kellynauert ! Lots of great improvements

**Review by hbradio — APPROVED** [review]

**Pull request merged**